### ISSUE TITLE
`policy.payoutMaxAmount` may change after creation of first claim

### ISSUE THREAD

**Issue by rapidddenis** [etherisc/gif-contracts#107]
In confirmClaim

https://github.com/etherisc/gif-contracts/blob/b58fd27a34452e062d10b175f3bdd7ee16b9d1a7/contracts/modules/PolicyController.sol#L310-L314

`policy.payoutMaxAmount` may be decreased in-between creation and confirmation of the claim thus enforcing `claim.claimAmount > claim.confirmedAmount`

**Comment by matthiaszimmermann** [comment]
good catch, thank you. indeed using `adjustPremiumSumInsured` the `payoutMaxAmount` can be decreased below a value that makes sense from existing confirmed claim amounts. this is a state that the framework needs to avoid.
